src/components/ArrayFilter/ArrayFilter.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './style.css';
+
+const ArrayFilter = (props) => {
+  const { filterList } = props;
+
+  const handleChangeFilter = (type) => {
+    const buttons = document.querySelectorAll('.filter-item');
+    buttons.forEach(item => {
+      if(item.id === type){
+        item.classList.add('active');
+      } else {
+        item.classList.remove('active');
+      }
+    })
```

Comment:
Are we allowed to use the `document` object of JS in React? Is there any better way to do this by making use of React's fundamentals?

Reply:
I fixed this by adding useRef hook